include/tracing.h

```diff
@@ -21,6 +21,7 @@
 #if CONFIG_TRACING
 void z_sys_trace_idle(void);
 void z_sys_trace_isr_enter(void);
+void z_sys_trace_isr_exit(void);
 void z_sys_trace_isr_exit_to_scheduler(void);
```

Comment:
Is this now used?

Reply:
Yes, it is used for example for ARM architecture.